api_test.go

```diff
@@ -78,11 +78,12 @@ func getTokenResponse() *tokenResponse {
 	return &tokenResponse{
 		Messages: []string{},
 		Token:    "tok-" + randString(20),
+		ID:       1,
 	}
 }
 
-func getJSONRequestMap(r *http.Request) map[string]string {
-	var requestJson = map[string]string{}
+func getJSONRequestMap(r *http.Request) map[string]interface{} {
+	var requestJson = map[string]interface{}{}
```

Comment:
You don't need to create an instance here as Decode will do that. Just `var requestJson map[string]interface{}` is fine.